Automattic/jetpack
# Update/social move notices near connections

Fixes https://github.com/Automattic/jetpack-reach/issues/495

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/495
## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the post editor, turn off sig and have no media, possibly also have a broken connection.
* Validate the notice placement

![CleanShot 2024-08-08 at 18 21 08 png](https://github.com/user-attachments/assets/8a1d0741-e5f1-4a44-be1e-c492f2f1fe49)
